Makefile

```diff
@@ -0,0 +1,27 @@
+
+.PHONY: verify-glide-installation install-vendor update-vendor all install check
+
+all:
+	go build github.com/giantswarm/k8scloudconfig
+
+verify-glide-installation:
+	@which glide || go get github.com/Masterminds/glide
+	@which glide-vc || go get github.com/sgotti/glide-vc
+
+install-vendor: verify-glide-installation
+	glide install --strip-vendor
+	glide-vc --use-lock-file
+
+update-vendor: verify-glide-installation
+	glide update --strip-vendor
+	glide-vc --use-lock-file
+
+verify-go-bindata-installation:
+	@which go-bindata || go get -u github.com/jteeuwen/go-bindata/...
+
+update-bindata: verify-go-bindata-installation
```

Comment:
It's used manually by someone who edits the template files.

What we may check in CircleCI is whether bindata is up to date (so probably generate bindata and compare with the bindata.go file which is commited), but I think we shouldn't do it in this PR - it's already huge..